runtime/browser/web_application.cc

```diff
@@ -485,13 +484,74 @@ void WebApplication::Terminate() {
   }
 }
 
+void WebApplication::CreateWindow(common::AppControl* appcontrol) {
+  auto window_category = GetWindowCategory(appcontrol);
+  window_.reset(CreateNativeWindow(window_category));
```

Comment:
Why always reset the window, 
If same window category was used, no need to create window